File: 351004/Student/Lab3/discussion/src/main/java/com/example/task310/entity/Post.java
package com.example.task310.entity;

import org.springframework.data.cassandra.core.mapping.Column;
import org.springframework.data.cassandra.core.mapping.PrimaryKey;
import org.springframework.data.cassandra.core.mapping.Table;
import lombok.Data;

@Data
@Table("tbl_post")
public class Post {

    @PrimaryKey
    private Long id;

    @Column("country")
    private String country;

    @Column("issue_id")
    private Long issueId;

    @Column("content")
    private String content;
}

File: 351002/vorobei/Discussion/DataAccess/Models/Post.cs
using Cassandra.Mapping.Attributes;

namespace DataAccess.Models
{
    [Table("posts")]
    public class Post : BaseEntity
    {
        [Column("story_id")]
        public int StoryId { get; set; }

        [Column("content")]
        public string Content { get; set; } = string.Empty;
    }
}
